build.gradle

```diff
@@ -30,8 +32,24 @@ dependencies {
 	runtimeOnly 'org.postgresql:postgresql'
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation 'org.junit.jupiter:junit-jupiter-api:5.7.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.2'
```

Comment:
czy oby na pewno nalezy dodać junit do runtime? (może tak, pytam tylko czy to nie przypadek)

Reply:
Faktycznie nie robi to różnicy, pospiesznie dodałem